Formula/cookiecutter.rb

```diff
@@ -101,6 +101,8 @@ def install
   end
 
   test do
+    ENV["LC_ALL"] = "en_US.UTF-8"
+    ENV["LANG"] = "en_US.UTF-8"
```

Comment:
The cookiecutter CLI uses [click](http://click.pocoo.org/) under the hood, and this library requires these 2 environment variables to be set for Surrogate Handling: 

http://click.pocoo.org/5/python3/#python-3-surrogate-handling

I initially omitted it, and the build failed with the `RuntimeError` from the link above.